### PR TITLE
Structured log events for bootstrap, requires Akka 2.5.27, #629

### PR DESCRIPTION
**Pull request by patriknw**
Refs #629

### CODE REVIEW

File: integration-test/kubernetes-api-java/pom.xml [hunk 1]
```diff
@@ -18,9 +18,9 @@
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
     <encoding>UTF-8</encoding>
-    <akka.version>2.5.19</akka.version>
+    <akka.version>2.5.27</akka.version>
```
Comment (patriknw) [inline integration-test/kubernetes-api-java/pom.xml]: travis failed with "Current version of Akka is [2.5.19], but cluster-bootstrap requires version [2.5.27]", so updated this. Not sure if there are more places but didn't find anything more when searching.

File: integration-test/kubernetes-api-java/pom.xml [hunk 2]
```diff
@@ -18,9 +18,9 @@
...
     <akka.http.version>10.1.6</akka.http.version>
-    <akka-management.version>0.20.0</akka-management.version>
+    <akka-management.version>1.0.5</akka-management.version>
```
Comment (patriknw) [inline integration-test/kubernetes-api-java/pom.xml]: should this be updated? 0.20.0 seems very old?

Comment (raboof) [reply integration-test/kubernetes-api-java/pom.xml]: yes, ideally we should pass this in via a property instead of having it as a constant in the pom.xml

**Comment by patriknw** [comment]
rebased

**Pull request merged**